src/stable/easegress/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v2
+name: easegress
+type: application
+version: 1.0.0
```

Comment:
Can you add a maintainer to this chart?

Reply:
thanks a lot for your reply! i will update it as soon as possible!

Reply:
Hi, sorry for delay... I have add the maintainers for easegress.... please check...